app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/index.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { TableBody, Checkbox, Divider, Box } from '@material-ui/core';
+import React from 'react';
+import T from 'i18n-react';
+import TableRowWidget from 'components/WranglerGrid/SelectColumnPanel/DataTable/TableRow';
+import { IColumnTableProps } from 'components/WranglerGrid/SelectColumnPanel/DataTable/types';
+import {
+  ADD_TRANSFORMATION_PREFIX,
+  SELECT_COLUMN_LIST_PREFIX,
+} from 'components/WranglerGrid/SelectColumnPanel/constants';
+import { MULTI_SELECTION_COLUMN } from 'components/WranglerGrid/SelectColumnPanel/constants';
+import { TableContainer, Table, TableHead, TableRow, TableCell } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import styled from 'styled-components';
+import { NormalFont, SubHeadBoldFont } from 'components/common/TypographyText';
+import { NoDataSVG } from 'components/GridTable/iconStore';
+
+const SelectColumnTableContainer = styled(TableContainer)`
+  height: 100%;
+`;
+
+const SelectColumnTable = styled(Table)`
+  display: flex;
+  flex-direction: column;
+`;
+
+const SelectColumnTableHead = styled(TableHead)`
+  height: 54px;
+`;
+
+const SelectColumnTableRow = styled(TableRow)`
+  font-weight: 400;
```

Comment:
how similar is this styling to the other table row?

Reply:
Updated it and made it reusable